source/_integrations/device_tracker.mqtt.markdown

```diff
@@ -232,7 +232,7 @@ value_template:
 
 ### Using the discovery protocol
 
-The device_tracker can be created via a discovery topic that follows the following topic name convention: `<discovery_prefix>/device_tracker/[<node_id>/]<object_id>/config`.
```

Comment:
This change looks incorrect to me, see also:

<https://github.com/home-assistant/core/blob/416aad32cc6607271b00e210360ae75ee339dc04/homeassistant/components/mqtt/discovery.py#L40-L43>

Could you point me to resources pointing out that the existing documentation was incorrect and your change is the right one?

../Frenck

Reply:
Thanks for that link.
Because of it I found https://www.home-assistant.io/docs/mqtt/discovery/ and that does talk about how node_id is optional (which is why the square brackets).

Instead, I’d like to propose changing the documentation to add details about node_id (there are currently no other mentions on the device tracker page), and to link to the discovery documentation.

Would that be better as a separate PR, or as a modification to this PR?

Reply:
Feel free to extend this PR (as in its current form, it is incorrect as it seems).